src/main/java/duke/gui/MainWindow.java

```diff
@@ -0,0 +1,55 @@
+package duke.gui;
+
+import duke.Duke;
+import duke.Ui;
+import javafx.fxml.FXML;
```

Comment:
It would be better to separate different categories of imports with a line break.